i18n.php

```diff
@@ -684,19 +700,19 @@
   ),
   'ka' => 
   array (
-    'and' => 'და<|*',
+    'and' => 'ასევე|და|*',
```

Comment:
This change is BC. It adds support for a new translation (ასევე) and removes the trailing `<` from the existing translation.

The `<` is handled here: https://github.com/Behat/Gherkin/blob/e07986ef3e9b6621dcd7069831d96fef9c76d4ac/src/Lexer.php#L342-L344

So the old translation with `<` would have been converted to `და\s+` while the new is `და\s*`. This will therefore still match any existing scenarios where users have added a whitespace after the keyword, they just won't be required to add a whitespace in future.